apps/music/js/ui/title_bar.js

```diff
@@ -43,6 +59,17 @@ var TitleBar = {
     ModeManager.pop();
   },
 
+  showScanProgress: function(info) {
+    this.scanProgress.classList.remove('hidden');
+    this.scanCount.textContent = info.count;
+    this.scanArtist.textContent = info.artist || '';
+    this.scanTitle.textContent = info.title || '';
```

Comment:
We'd only need "Unknown artist", since we don't show the album. I was just keeping the behavior the same as it is now; we could have UX look into it in another bug, but I think they wanted to make bigger changes to this...